local_pathfinding/node_navigate.py

```diff
@@ -175,8 +178,22 @@ def get_desired_heading(self) -> float:
             self.gps, self.ais_ships, self.global_path, self.filtered_wind_sensor, self.planner
         )
 
-        # TODO: create function to compute the heading from current position to next local waypoint
-        return 0.0
+        current_waypoint = self.gps.lat_lon
+        next_waypoint = None
+        waypoints = self.local_path.waypoints
+        for i in range(len(waypoints)):
+            if waypoints[i] == current_waypoint and i < len(waypoints) - 1:
+                next_waypoint = waypoints[i + 1]
+                break
+        if next_waypoint is None:
+            return 0.0
+        heading = GEODESIC.inv(
```

Comment:
I think `GEODESIC.inv()` has multiple returns ([documentation](https://pyproj4.github.io/pyproj/stable/api/geod.html#pyproj.Geod.inv:~:text=or%20array%20%E2%80%93-,Forward%20azimuth(s),-(azi12))), and I think you only want the forward azimuth (image at bottom)

I believe Patrick is a fan of this kind of syntax for this situation: `_, value, _ = myfunc()`, where `myfunc()` returns a touple:

```
def myfunc():
	return (1, 2, 3)
```
In this example, `print(value)` outputs 2.


![image](https://github.com/UBCSailbot/local_pathfinding/assets/71156197/670a53f4-5927-4034-8214-0410b495845a)